build_lang.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+if [ $3 == src/lang_rev.awk ]; then
+cat > $1 <<-EOF
+/*
+ * code automatically generated with make and build_lang.sh
+ */
+
+#ifndef LANG_REV_H
+#define LANG_REV_H
+#include "language.h"
+static inline language_code
+lang_from_str(const char *str)
```

Comment:
very minor: for sake of coherent naming, I'd propose to call this function `language_from_str`.